tlatools/org.lamport.tlatools/src/tla2sany/xml/XMLExporter.java

```diff
@@ -160,8 +153,13 @@ public static final void main(String[] args) throws XMLExportingException {
       // validate the file, do not fail if there is a URL connection error
       if (!offline_mode) { //skip validation in offline mode
         try {
-          SchemaFactory factory = SchemaFactory.newInstance(W3C_XML_SCHEMA);
-          Schema schema = factory.newSchema(new URL(TLA_SCHEMA));
+          SchemaFactory factory = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI);
+          URL schemaFile = XMLExporter.class.getResource("sany.xsd");
+          if (null == schemaFile) {
+            ToolIO.err.println("ERROR: Unable to find sany.xsd schema file embedded in jar.");
```

Comment:
"...that is expected to be embedded in the tla2tools.jar"?

Reply:
That does read a bit better but I think if https://github.com/tlaplus/tlaplus/issues/1048 comes to pass we can't assume the jar will be called tla2tools.jar.

Reply:
"...that is expected to be embedded in the jar"?